native-libs/deps/recipes/mymonerocorecpp/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.4.1)
 
-SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -fvisibility=hidden -fvisibility-inlines-hidden")
```

Comment:
We compile boost with those settings so we do the same with `mymonerocorecpp` which uses it to avoid linker warnings about visibility mismatch.